rust/src/graph/pathfinding.rs

```diff
@@ -28,7 +28,8 @@ pub fn find_reach(
     &seen.into_iter().collect::<FxHashSet<_>>() - from_modules
 }
 
-pub fn find_shortest_path_bidirectional(
+/// Finds the shortest path, via a bidirectional BFS.
+pub fn find_shortest_path(
```

Comment:
🍷 Sneaking this little change into the PR - it's not related to the rest of the changes, I can remove it if you like.